packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details/class_details.dart

```diff
@@ -28,10 +35,35 @@ class HeapClassDetails extends StatelessWidget {
       );
     }
 
-    return RetainingPathTable(
+    final area1 = RetainingPathTable(
       entries: theEntries,
       selection: selection,
       isDiff: isDiff,
+      className: className!,
+    );
+
+    final area2 = ValueListenableBuilder<StatsByPathEntry?>(
+      valueListenable: selection,
+      builder: (_, selection, __) {
+        if (selection == null) {
+          return const Center(
+            child: Text(
+              'Select a group of instances to see the retaining path here.',
+            ),
+          );
+        }
+
+        return RetainingPathView(
+          path: selection.key,
+          controller: pathController,
+        );
+      },
+    );
+
+    return Split(
+      axis: Axis.horizontal,
+      initialFractions: const [0.7, 0.3],
+      children: [area1, area2],
```

Comment:
Nit: rename these variables to something more descriptive. Maybe `retainingPathView` and `statsView`?